app/domain/authentication/usecase/authentication.go

```diff
@@ -0,0 +1,18 @@
+package usecase
+
+import (
+	"github.com/patriciapedrosaa/transfer-me/app/domain/account"
+	"github.com/patriciapedrosaa/transfer-me/app/domain/authentication"
+)
+
+type Authentication struct {
+	authenticationRepository authentication.Repository
+	accountRepository        account.Repository
+}
+
+func NewTransferUsecase(authenticationRepository authentication.Repository, accountRepository account.Repository) Authentication {
```

Comment:
I guess we have a typo here, shouldn't be NewAuthenticationUseCase?

Reply:
Yes! adjusted